examples/checkout/sdk-load-widgets-with-nextjs/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@examples/sdk-load-widgets-with-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prepare:widgets": "mkdir -p tests/utils/local-widgets-js/ && cp $(yarn workspace @imtbl/checkout-widgets exec pwd)/dist/*.js tests/utils/local-widgets-js/",
+    "test": "USE_LOCAL_BUNDLE=true playwright test playwright test",
+    "test:latest": "USE_LOCAL_BUNDLE=true INTERCEPT_WIDGETS_VERSION=latest playwright test",
+    "test:ui": "playwright test --ui",
```

Comment:
Can you set up the "test" command to test everything you want on the examples workflow? This way we can keep the tests handling to the examples package.jsons test scripts, rather than having the workflows specify all the test commands needed to be run.

For eg, testing local_bundle and intercept_widgets=latest, you could add a new test:local script and do something like "test": "yarn test:local && yarn test:latest"?

Reply:
Hmm probably yes, it actually probably just makes sense to have a single:

```
"test": "USE_LOCAL_BUNDLE=true  INTERCEPT_WIDGETS_VERSION=latest playwright test playwright test",
```

Reply:
In which case I can just set those as defaults and we don't need to pass in env variables.